### DISCUSSION TITLE
Use last version number somewhere in the app

### DISCUSSION THREAD

**Discussion by benoitgrasset** [semantic-release/semantic-release#2991] [Q&A]
Hello,

I've set up two CI jobs in GitHub Actions:

- The first job is to trigger **Semantic Release**. Its purpose is to update my version tag, generate a changelog, and create a new release.

- The second job is to build my project using Docker. These two jobs are sequential.

Ideally, I'd like to display my new version number on my login page. This version number is generated during the CI process and needs to be "sent" or "shared" with my app just before the build to ensure it has the correct corresponding number.

What would be the best way to proceed?

Here is my `release.config.js` file:

```
module.exports = {
  branches: ['main'],
  tagsFormat: 'v${version}',
  plugins: [
    '@semantic-release/commit-analyzer', // Analyzes commits to determine the version bump
    '@semantic-release/release-notes-generator', // Generates release notes based on commits
    [
      '@semantic-release/changelog', // Updates the changelog file with the new release version
      {
        changelogFile: 'CHANGELOG.md',
      },
    ],
    '@semantic-release/npm',
    '@semantic-release/github',
    [
      '@semantic-release/git',
      {
        assets: ['CHANGELOG.md', 'package.json'],
      },
    ],
  ],
};
```

**Comment by seebeen** [comment]
If you have two CI jobs you can use `@semantic-release/exec` to do something like this
```bash
echo ${nextRelease.version} > /tmp/VERSION
```

And then use artifact-upload and artifact-download to get the version string from the file.

From there, do as you wish 😊
